components/collator/tests/tests.rs

```diff
@@ -1741,7 +1741,7 @@ fn test_conformance_non_ignorable() {
             if line.is_empty() {
                 continue;
             }
-            if line.starts_with(&[b'#']) {
+            if line.starts_with(b"#") {
```

Comment:
here too

Reply:
Nope: it doesn't work. `starts_with` only accepts Patterns in the case of strings, not slices

Reply:
but it used a pattern beofre?

Reply:
No, it used a slice.

Reply:
slices happen to be Patterns, however this function accepts slices only

Reply:
not sure this is an improvement